### DISCUSSION TITLE
Public announcement of Reqnroll, the v1.0.0 framework and the v2024.1 of the Visual Studio extension

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#9] [Announcements]
- [General announcement, motivation, goals](https://reqnroll.net/news/2024/02/from-specflow-to-reqnroll-why-and-how/)
- [Reqnroll v1.0 announcement](https://reqnroll.net/news/2024/02/reqnroll-v1-0-released-with-specflow-compatibility/)
- [Reqnroll Visual Studio v2024.1 announcement](https://reqnroll.net/news/2024/02/reqnroll-visual-studio-extension-v2024-1-released/)

**Comment by Vic152** [comment]
Hey Gaspar,

It takes a lot of courage to do what you did. Well done, and thank you.

Vic
